app/grandchallenge/serving/views.py

```diff
@@ -74,3 +75,33 @@ def serve_submissions(request, *, submission_pk, **_):
         )
 
     raise PermissionDenied
+
+
+def serve_component_interface_value(
```

Comment:
Could we get some tests on this function? It's really annoying to test but it's important when dealing with permissions. See https://github.com/comic/grand-challenge.org/blob/24ea0061be90a9fca3ca093d847e206052e9e7b0/app/tests/serving_tests/test_views.py#L16

Reply:
Yes, hadn't committed the test yet.

Reply:
Ah! Sorry, didn't notice this was a draft. My fault.

Reply:
I changed it to a draft, should have done that right away. Got distracted...